lib/shared/information.js

```diff
@@ -0,0 +1,98 @@
+/*
+ * Copyright 2016 resin.io
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+'use strict';
+
+const _ = require('lodash');
+const semver = require('semver');
+const packageJSON = require('../../package.json');
+
+/**
+ * @summary Application release types
+ * @namespace RELEASE_TYPE
+ * @public
+ */
+exports.RELEASE_TYPE = {
+
+  /**
+   * @property {String} PRODUCTION
+   * @memberof RELEASE_TYPE
+   * @description
+   * Production release type
+   */
+  PRODUCTION: 'PRODUCTION',
+
+  /**
+   * @property {String} SNAPSHOT
+   * @memberof RELEASE_TYPE
+   * @description
+   * Snapshot release type
+   */
+  SNAPSHOT: 'SNAPSHOT',
+
+  /**
+   * @property {String} UNKNOWN
+   * @memberof RELEASE_TYPE
+   * @description
+   * Unknown release type
+   */
+  UNKNOWN: 'UNKNOWN'
+
+};
+
+/**
+ * @summary Get the release type from a version string
+ * @function
+ * @public
+ *
+ * @param {String} version - application version
+ * @returns {RELEASE_TYPE} release type
+ *
+ * @example
+ * const version = require('../../package.json');
+ * const releaseType = information.getReleaseType(version);
+ *
+ * if (releaseType === information.RELEASE_TYPE.PRODUCTION) {
+ *   console.log('This is a production release!');
+ * }
+ */
+exports.getReleaseType = (version) => {
```

Comment:
I rather have a separate `getReleaseTypeFromCurrentVersion` or something like that, just to keep this function simple, but I guess that might not be necessary if we create a facade function like recommended in https://github.com/resin-io/etcher/pull/1183#discussion_r109281158.